### ISSUE TITLE
Convert all cells to text format

### ISSUE THREAD

**Issue by mhavelka77** [exceljs/exceljs#1642]
**Hi!**

I have hopefully a quick question. Is somehow possible to convert all cell of a worksheet to Excel **text** format? I'm importing a document from csv and I tried the map option to always return string values, however when I afterwards open the sheet in Excel, it still processes all cells in their "General" format. 

Here is my current code:

```
const exceljs = require("exceljs");
const { Readable } = require("stream");

let source = Readable.from([msg.payload]);

const options = {
    map(value, index) {
        return value.toString();
    }
};

const workbook = new exceljs.Workbook();
try {
    workbook.csv.read(source, options)
    .then(() => {
    workbook.xlsx.writeBuffer()
        .then((value) => {
            // save value to file...
        }).catch((err) => {
            throw err; 
        })
    })
    .catch((err) => {
       throw err; 
    });
}

catch (err) {
    console.err(err);
}

return;
```

Thank you in advance!

**Comment by ihms97** [comment]
Take a look at [this](https://github.com/exceljs/exceljs/issues/917)
hope it help after few weeks

**Comment by mhavelka77** [comment]
Solved here: #917
